Vwitter/Models/GroupMembership.h

```diff
@@ -0,0 +1,19 @@
+//
+//  GroupMembership.h
+//  Vwitter
+//
+//  Created by Christina Li on 7/12/22.
+//
+
+#import <Parse/Parse.h>
+#import "GroupDetails.h"
+NS_ASSUME_NONNULL_BEGIN
+
+@interface GroupMembership : PFObject<PFSubclassing>
+
+@property (strong, nonatomic) GroupDetails *groupId;
```

Comment:
nit: Same here, since this is a pointer to the actual group, I would just call it `group`